uwblueprint/starter-code-v2
# Prisma Migration - User Service

## Notion ticket link
[Migrate-User-Service-to-Prisma](https://www.notion.so/uwblueprintexecs/Migrate-User-Service-to-Prisma-4e6f3e6025fa4a2381c01a1b7df899e0?pvs=4



## Implementation description
* Migrating the user service interface and implementation to implement Prisma in place of Sequelize.



## Steps to test
1. Re-create the repository starter-code-cli.
2. Ensure the user routes are functional with Postman.



## What should reviewers focus on?
* 


## Checklist
